unpwn.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "bloomer", "~> 1.0"
-  spec.add_dependency "pwned", "~> 1.2"
+  spec.add_dependency "pwned", "~> 2"
```

```suggestion
  spec.add_dependency "pwned", "~> 2.0"
```

The ~> operator allows the last given number to change. We want to use 2.x, so that means we need to include the minor version to allow it to change.